citi-onboarding/DADe-onepage
# feat(feature/create-contact-backend-connection): added contact links backend connection

## What I did
- Added backend connection to the contact view (editable links)

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   1. `git checkout develop`
   2. `git pull`
   3. `git checkout feature/create-contact-backend-connection`
   4. `git pull origin feature/create-contact-backend-connection`
   5. `npm install`
   6. `cd client/`
   7. `npm start`

3. Open another terminal and run Keystone server

4. In your browser go to http://localhost:3000 (should open automatically when running `npm start`)

5. Check if the view component (Contact) looks just like the mockup in different screen sizes (minimum width: 1366px) and if when you increase it past 1920px it remains the same size (Doesnt grow along with the screen).

6. In your browser, go to http://localhost:3001/admin/contatos

7. Change the text (link) of the list item (every field), and see if it displays correctly after your changes and if the changes actually happen, click the text and see if it takes you to the corresponding link.

8. After your changes, make sure to return the list item to its original state (corresponding link to the social media website).

9. Read the changed files section on GitHub 

10. Test the things that you think are worth testing, even the ones that are not in this description
